opengapps-packages.mk

```diff
@@ -28,8 +28,12 @@ GAPPS_PRODUCT_PACKAGES += \
 endif
 
 ifneq ($(filter $(TARGET_GAPPS_VARIANT),nano),) # require at least nano
+# support only arm based platforms
+ifeq ($(TARGET_ARCH), $(filter $(TARGET_ARCH), arm arm64))
```

Comment:
https://github.com/opengapps/opengapps/blob/master/scripts/inc.buildtarget.sh#L280

The following check emulates the logic in `inc.buildtarget.sh` more closely:

```makefile
ifneq ($(filter arm%,$(TARGET_ARCH)),) 
...
endif
```
But, this is a minor suggestion, so feel free to ignore it :-)